README.md

```diff
@@ -59,13 +62,11 @@ Finally, you will also have to make sure that you are ignoring files that need i
 
 ### Checkout dependencies
 
-App Inventor uses Blockly, the web-based visual programming editor from Google, as a core part of its editor. Blockly core is made available to App Inventor as a git submodule. The first time after forking or cloning the repository, you will need to perform the following commands:
+App Inventor uses the [Closure compiler](https://developers.google.com/closure/compiler) and the [Picrin](https://picrin.readthedocs.io/en/latest/) Scheme implementation. It is unlikely that most contributors will need to make changes to these dependencies, but if so, you can initialize and track these libraries as submodules. The first time after forking or cloning the repository, you can perform the following commands:
 
     $ git submodule update --init
 
-For developers who will be working on Blockly within the context of App Inventor, the preferred checkout procedure is to perform a `git submodule init`, edit the `.git/config` file to use the read/write SSH URL for [MIT CML's Blockly fork](https://github.com/mit-cml/blockly) instead of the public read-only HTTPS URL assumed by default (to support pushing changes). After changing `.git/config`, a `git submodule update` will pull the repository.
```

Comment:
Even though we don't use the submodule anymore, ut would be nice to keep around the information that we currently use a (minimally) forked version of Blockly that lives at https://github.com/mit-cml/blockly and is currently in the rc/10.5.0 branch of that repo.